OMCompiler/Compiler/Util/Flags.mo

```diff
@@ -853,27 +853,8 @@ constant ConfigFlag CHEAPMATCHING_ALGORITHM = CONFIG_FLAG(13, "cheapmatchingAlgo
     Util.gettext("Sets the cheap matching algorithm to use. A cheap matching algorithm gives a jump start matching by heuristics."));
 
 constant ConfigFlag MATCHING_ALGORITHM = CONFIG_FLAG(14, "matchingAlgorithm",
-  NONE(), EXTERNAL(), STRING_FLAG("PFPlusExt"),
```

Comment:
We should have a look at jens' paper where he compared all these matching algorithms. Maybe there are special cases which we are not considering right now. If not this seems superfluous.